.travis.yml

```diff
@@ -25,19 +25,10 @@ install:
     - if [ "$CHECKS" = "yes" ]; then travis_retry composer require --no-interaction fabpot/php-cs-fixer:^1.11; fi;
```

Comment:
as moved phpcs and phpmd to check_styles.sh, the tools itself may be installed only when with $CHECKS